checkout-ui-custom/src/partials/Deliver/Addresses.js

```diff
@@ -24,12 +30,19 @@ const Addresses = () => {
       console.error('ERROR getAddresses', e);
       throw new Error('Error getAddresses', e.message);
     });
+};
 
-  return `
- <div class="bash--addresses shimmer" id="bash-address-list">
-    Loading address book...
-  </div>
-  `;
+const Addresses = () => {
+  RefreshAddressList();
```

Comment:
There is a high chance that using the index DB methods to get Address's will mean we dont have to do this